server/src/main/java/org/opentosca/toscana/plugins/scripts/BashScript.java

```diff
@@ -0,0 +1,62 @@
+package org.opentosca.toscana.plugins.scripts;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import org.opentosca.toscana.core.plugin.PluginFileAccess;
+
+import org.apache.commons.io.FileUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class BashScript {
+    private final static Logger logger = LoggerFactory.getLogger(BashScript.class);
+    private final String scriptsTargetDir = "content/scripts/";
+    private String name;
+    private PluginFileAccess access;
+    private File script;
+
+    public BashScript(PluginFileAccess access, String name) throws IOException {
```

Comment:
Please add documentation for parameters. Noone knows that you append ".sh" to the name in the implementation.